provision-contest/ansible/EOC/README.md

```diff
@@ -0,0 +1,63 @@
+## DOMjudge EOC ansible helper
+This ansible playbook should automate must of the tasks DOMjude needs to perform for the End Of Contest procedures during the ICPC WF's. It was created for the WF 2021 in Dahka, which took place in 2022.
+
+Two types of authentication are used and supported. Session based auth and basic auth. 
+ - Session based auth retrieves a CSRF token and fakes a login action.
+ - Basic auth headers are constructed and used. This is to fully mitigate [annoying basic-auth issues](https://askubuntu.com/questions/1070838/why-wget-does-not-use-username-and-password-in-url-first-time/1070847#1070847), even though the `force_basic_auth` option should already prevent these issues from arising.
+
+The full playbook can be run using the following command: `ansible-playbook main.yml --extra-vars "@facts.yml" --tags all`. See the setup section to see what the contents of facts.yml are supposed to be.
```

Comment:
Super-nit: move the command to an extra line you it's even easier to copy/ paste.